src/store/ducks/opensrp/hierarchies/index.ts

```diff
@@ -297,6 +301,55 @@ export const getNodeById = () =>
     return nodeOfInterest;
   });
 
+/**
+ * Get the root id given a node
+ *
+ * Note: If you pass in rootJurisdictionId then that is what will be returned
+ * You should pass in rootJurisdictionId as '' when don't know it
+ *
+ * @param state - the store
+ * @param props -  the filterProps
+ */
+export const getRootByNodeId = () =>
+  createSelector(
+    [getTreesByIds, getNodeId, getRootJurisdictionId],
+    (trees, nodeId, rootJurisdictionId) => {
+      if (rootJurisdictionId && rootJurisdictionId !== '') {
```

Comment:
can we use `!!rootJurisdictionId` here instead?

Reply:
Done